client/src/modules/vouchers/templates/action.cell.html

```diff
@@ -39,6 +39,11 @@
         <span class="fa fa-file-text-o"></span> <span translate>REPORT.VIEW_VOUCHER</span>
       </a>
     </li>
+    <li ng-show="row.entity.type_id === 12 || row.entity.type_id === 13 || row.entity.type_id === 14">
```

Comment:
In the future, it would be cleaner (and probably faster) to just stick an `isStockMovement` to these rows and move this logic into the JS controller.  I'm thinking something like:
```js
const stockMovementTypes = [
  bhConstant.transactionTypes.STOCK_EXIT,
  bhConstant.transactionTypes.STOCK_ENTRY,
  // .. etc
];

rows.forEach(row => {
  row.isStockMovement = stockMovementTypes.includes(row.type_id);
});
```

This isn't a critical change though.